src/hostname/hostnamed.c

```diff
@@ -212,7 +212,7 @@ static const char* fallback_chassis(void) {
            unreliable enough, so let's not do any additional guesswork
            on top of that.
 
-           See the SMBIOS Specification 4.0 section 7.4.1 for
+           See the SMBIOS Specification 3.0 section 7.4.1 for
```

Comment:
Well, the URL to the PDF below references version 3.0, so you should either update the link as well or leave the comment as it is.